content/browser/android/content_startup_flags.cc

```diff
@@ -69,6 +69,12 @@ void SetContentCommandLineFlags(bool single_process) {
     base::CommandLine::ForCurrentProcess()->AppendSwitchASCII(
         switches::kDisableFeatures, disabled_features);
   }
+#endif
+#if defined(SERVICE_OFFLOADING)
+  if (base::ServiceOffloading::IsEnabled()) {
```

Comment:
I think it is better to move this code to the bottom of the function because the 'SERVICE_OFFLOADING' condition is checked at the bottom as well.

Reply:
Done